markdown/4.x/en/api/ff-communication.api.md

```diff
@@ -1,11 +1,11 @@
 ## `ff-communication`
 ___
-### Properties
+### Properties (`*` - required)
```

Comment:
Shouldn't `url` and `channel` be marked as required too?

Reply:
I think so. I left them out here because they are somewhat unrelated nd there is a branch specifically for the `*`. Clarifying.

Reply:
Generically added the "required" markers and rebuilt the history.